packages/data-point/lib/stores/entity-types.test.js

```diff
@@ -26,33 +13,24 @@ it('setup', () => {
 
 it('source#getStore', () => {
   const result = entityTypes.getStore()
-  expect.assertions(1)
   expect(_.isObject(result)).toBeTruthy()
 })
 
 it('source#add/get', () => {
   entityTypes.add('foo', FooEntityType)
-
-  const foo = entityTypes.get('foo')
-
-  expect.assertions(2)
-  expect(_.isFunction(foo.create)).toBeTruthy()
-
-  const doNotAllowOverride = _.attempt(entityTypes.add, 'foo', {})
+  const doNotAllowOverride = _.attempt(entityTypes.add, 'foo', () => {})
   expect(_.isError(doNotAllowOverride)).toBeTruthy()
```

Comment:
do `expect(doNotAllowOverride).toBeInstanceOf(Error)`? more concise IMO

Reply:
yeah, its legacy code before jest, we could refactor it, was not part of the scope of the change